paper/proposal.md

```diff
@@ -0,0 +1,16 @@
+# Research Paper Proposal
+
+## Title: A potential key to open the chamber of secrets of neurocognitive functions by applying Thunder---a big data tools in neuroscience field
```

Comment:
wait-- I thought the Chamber of Secrets was a bad thing!?

Reply:
Thanks Professor! 
I can change the Chamber of Secrets in the title. Maybe write it as "open the door of", if that is OK. 
Thanks again.

Reply:
I was only making a Harry Potter joke! Call it whatever you like